client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/index.tsx

```diff
@@ -16,14 +15,24 @@ import { setStore } from 'calypso/state/redux-store';
 import Launchpad from '../index';
 import { buildSiteDetails, defaultSiteDetails } from './lib/fixtures';
 
-const mockUseQuery = () => () => ( { isLoading: true, data: null } );
-
 jest.mock( '../launchpad-site-preview', () => () => {
 	return <div></div>;
 } );
 
 jest.mock( 'react-query', () => ( {
-	useQuery: jest.fn(),
+	useQuery: () => ( { isLoading: true, data: null } ),
```

Comment:
`NB`: Do we still need to mock this if we're mocking `useGetDomainsQuery`?

Reply:
Good catch. Its not needed anymore.